src/ORM/Connect/DBSchemaManager.php

```diff
@@ -314,7 +314,7 @@ public function transAlterField($table, $field, $schema)
      *
      * @param string $table Name of the table to hold this index
      * @param string $index Name of the index to update
-     * @param array $schema Already parsed index specification
+     * @param array|false $schema Already parsed index specification, or false to drop the index.
```

Comment:
This change might be contentious from a semver perspective. It's not _technically_ a breaking change according to our definition of public API, but any subclasses overriding this method (e.g. for postgres or mssql if they override this) are unlikely to be looking for `false` here.

Reply:
They'll still need to be updated to handle the `dropped` key - but you're right that this is a better solution as it's just "you may not get this feature" instead of "this breaks your shit" lol